repl/jupyter-lite.json

```diff
@@ -1,6 +1,11 @@
 {
     "jupyter-lite-schema-version": 0,
     "jupyter-config-data": {
+      "litePluginSettings": {
+        "@jupyterlite/pyodide-kernel-extension:kernel": {
+          "pyodideUrl": "https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js"
```

Comment:
In context of a tutorial, wouldn't it make more sense to pin to a definite version that we can test beforehand?

Reply:
See https://github.com/pyodide/pyodide/pull/5003#issuecomment-2304836546

Reply:
Oops, it looks like I was 'reviewing' instead of just commenting, so my comment (above) had been pending for 4 hours... :wink:

Reply:
```suggestion
          "pyodideUrl": "https://cdn.jsdelivr.net/pyodide/v0.27.0a2/full/pyodide.js"
```